timely_beliefs/beliefs/utils.py

```diff
@@ -445,10 +445,13 @@ def set_reference(
     if return_expected_value is True:
         reference_df = reference_df.for_each_belief(get_expected_belief)
 
+    belief_timing_col = (
+        "belief_time" if "belief_time" in reference_df.index.names else "belief_horizon"
```

Comment:
Not sure I understand exactly what is happening here. But "belief_horizon" is used as a name in the concatenated and returned dataframe (lines 458ff). Is that still accurate when we dropped it?

Reply:
It is. We first get rid of the belief timing column and then reconstruct our own, where we choose the horizon perspective explicitly.